package.json

```diff
@@ -10,7 +10,7 @@
     "server": "nodemon server/server.js",
     "client": "npm start --prefix client",
     "dev": "concurrently \"npm run server\" \"npm run client\"",
-    "build": "node server/database/build.js"
+    "build": "node server/database/build.js run"
```

Comment:
what is this `run` here at the end ?

Reply:
https://github.com/FACN4/Student-Project-TopCorner/blob/c3c9521e0b973ddedcba8c62ddfee26b720e490c/server/database/build.js#L60-L62

Reply:
The build.js file has the following code `if (process.argv[2] === 'run') {
  buildDatabase();
}`... The run is a key word..

Reply:
there are 0 reasons that you would need that @ghassanmas @BenNewman90 but we will roll with it, cause this pull request is slowly becoming a blocker

Reply:
It's purpose meant to differentiate when you want to use the build.js as module in testing verses when you want to use it as a self executed function.